### PR TITLE
Comply with pv-terms

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [x] Code changes are covered by tests
- ~New functions added to `__init__.py`~
- ~API.rst is up to date, along with other sphinx docs pages~
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog 
- [x] audit notebooks for variables that can be renamed

See #175.

### CODE REVIEW

File: rdtools/filtering.py [hunk 1]
```diff
@@ -31,13 +31,14 @@ def normalized_filter(normalized, low_cutoff=0.01, high_cutoff=None):
     return (normalized > low_cutoff) & (normalized < high_cutoff)
 
 
-def poa_filter(poa, low_irradiance_cutoff=200, high_irradiance_cutoff=1200):
+def poa_filter(poa_global, low_irradiance_cutoff=200,
+               high_irradiance_cutoff=1200):
```
Comment (kanderso-nrel) [inline rdtools/filtering.py:35]: Not sure if the cutoff names should be changed.  Maybe `poa_global_low`?

File: rdtools/filtering.py [hunk 2]
```diff
@@ -31,13 +31,14 @@ def normalized_filter(normalized, low_cutoff=0.01, high_cutoff=None):
     return (normalized > low_cutoff) & (normalized < high_cutoff)
```
Comment (kanderso-nrel) [inline rdtools/filtering.py:31]: `filtering.normalized_filter` currently has `normalized` as a parameter.  Change to `energy_normalized`?

File: rdtools/filtering.py [hunk 3]
```diff
@@ -50,10 +51,11 @@ def poa_filter(poa, low_irradiance_cutoff=200, high_irradiance_cutoff=1200):
         Boolean Series of whether the given measurement is within acceptable
         bounds.
     '''
-    return (poa > low_irradiance_cutoff) & (poa < high_irradiance_cutoff)
+    return ((poa_global > low_irradiance_cutoff) &
+            (poa_global < high_irradiance_cutoff))
 
 
-def tcell_filter(tcell, low_tcell_cutoff=-50, high_tcell_cutoff=110):
+def tcell_filter(temperature_cell, low_tcell_cutoff=-50, high_tcell_cutoff=110):
```
Comment (kanderso-nrel) [inline rdtools/filtering.py:58]: Same comment about cutoff names.  `temperature_cell_low`?

File: rdtools/filtering.py [hunk 4]
```diff
@@ -94,19 +97,19 @@ def clip_filter(power, quant=0.98):
...
         quantile filter.
     '''
-    v = power.quantile(quant)
-    return (power < v * 0.99)
+    v = power_ac.quantile(quant)
+    return (power_ac < v * 0.99)
 
 
-def csi_filter(measured_poa, clearsky_poa, threshold=0.15):
+def csi_filter(poa_global_meas, poa_global_sim, threshold=0.15):
```
Comment (kanderso-nrel) [inline rdtools/filtering.py:104]: I think this change reduces clarity, but it's consistent with pv-terms.

File: rdtools/normalization.py [hunk 5]
```diff
@@ -11,7 +11,7 @@ class ConvergenceError(Exception):
     '''Rescale optimization did not converge'''
     pass
 
-def normalize_with_expected_power(pv, expected_power, irradiance, pv_input='power'):
+def normalize_with_expected_power(pv, power_sim, irradiance, pv_input='power'):
```
Comment (kanderso-nrel) [inline rdtools/normalization.py:14]: Not sure what to do about this function.  pv-terms defines terms for specific quantities, but `pv` can be either power or energy.  I personally prefer `expected_power` over `power_sim`, but again it's consistent with pv-terms.  Also I missed `irradiance`, should be `poa_global`.

File: rdtools/normalization.py [hunk 6]
```diff
@@ -76,17 +77,17 @@ def pvwatts_dc_power(poa_global, P_ref, T_cell=None, G_ref=1000, T_ref=25,
     ----------
     poa_global : pd.Series
         Total effective plane of array irradiance.
-    P_ref : float
+    pmp_ref : float
```
Comment (kanderso-nrel) [inline rdtools/normalization.py:80]: `pmp_ref` is the closest term I found to mean array nameplate capacity.  Possible that I missed a better alternative.

File: rdtools/normalization.py [hunk 7]
```diff
@@ -76,17 +77,17 @@ def pvwatts_dc_power(poa_global, P_ref, T_cell=None, G_ref=1000, T_ref=25,
...
         Time series assumed to be same frequency as `poa_global`.
         If omitted, the temperature term will be ignored.
-    G_ref : float, default 1000
+    irradiance_ref : float, default 1000
         Reference irradiance at standard test condition [W/m**2].
-    T_ref : float, default 25
+    temperature_ref : float, default 25
         Reference temperature at standard test condition [degrees celsius].
-    gamma_pdc : float, default None
+    gamma_pmp : float, default None
```
Comment (kanderso-nrel) [inline rdtools/normalization.py:90]: `gamma_pmp_ref`?

File: rdtools/normalization.py (hunk 6 above)
Comment (mdeceglie) [reply rdtools/normalization.py:80]: `power_dc_rated` ?

File: rdtools/filtering.py (hunk 1 above)
Comment (cdeline) [reply rdtools/filtering.py:35]: I'm ok with keeping this poa.  poa_global isn't strictly required here.

File: rdtools/filtering.py (hunk 2 above)
Comment (cdeline) [reply rdtools/filtering.py:31]: I do think that 'normalized' is a little vague...

File: rdtools/filtering.py (hunk 3 above)
Comment (cdeline) [reply rdtools/filtering.py:58]: These changes look ok.

File: rdtools/filtering.py (hunk 4 above)
Comment (cdeline) [reply rdtools/filtering.py:104]: maybe 'poa_meas' and 'poa_clearsky' ?

File: rdtools/normalization.py (hunk 6 above)
Comment (cdeline) [reply rdtools/normalization.py:80]: pmp_ref works for me here.

Comment (cdeline) [reply rdtools/normalization.py:80]: looks like T_cell, G_ref and T_ref could be changed though?  'temperature_cell', 'poa_ref' and 'temperature_ref'?

File: rdtools/filtering.py [hunk 8]
```diff
@@ -72,18 +74,19 @@ def tcell_filter(tcell, low_tcell_cutoff=-50, high_tcell_cutoff=110):
         Boolean Series of whether the given measurement is within acceptable
         bounds.
     '''
-    return (tcell > low_tcell_cutoff) & (tcell < high_tcell_cutoff)
+    return ((temperature_cell > low_tcell_cutoff) &
+            (temperature_cell < high_tcell_cutoff))
 
 
-def clip_filter(power, quant=0.98):
+def clip_filter(power_ac, quant=0.98):
```
Comment (kanderso-nrel) [inline rdtools/filtering.py:81]:
```suggestion
def clip_filter(power_ac, quantile=0.98):
```

File: rdtools/filtering.py (hunk 4 above)
Comment (mdeceglie) [reply rdtools/filtering.py:104]: Let's go with `poa_global_measured` and `poa_global_clearsky`

File: rdtools/normalization.py (hunk 5 above)
Comment (mdeceglie) [reply rdtools/normalization.py:14]: let's go with `power_expected`

Comment (mdeceglie) [reply rdtools/normalization.py:14]: stick with `pv`

File: rdtools/normalization.py (hunk 6 above)
Comment (mdeceglie) [reply rdtools/normalization.py:80]: Let's stick with `power_dc_rated`. Change `irradiance_ref` to `poa_global_ref`, and `temperature_ref` to `temperature_cell_ref`

Comment (mdeceglie) [reply rdtools/normalization.py:80]: @cwhanse @wholmgren I see that `gamma_pdc` is used in pvlib, but is listed as deprecated in pv-terms. Is there a plan to change to `gamma_pmp_normalized` in pvlib?

File: rdtools/normalization.py (hunk 7 above)
Comment (mdeceglie) [reply rdtools/normalization.py:90]: @cwhanse @wholmgren I see that gamma_pdc is used in pvlib, but is listed as deprecated in pv-terms. Is there a plan to change to gamma_pmp_normalized in pvlib?

Comment (wholmgren) [reply rdtools/normalization.py:93]: I don't have any plans to change it.

Comment (cwhanse) [reply rdtools/normalization.py:93]: Agree with @wholmgrem here

Comment (mdeceglie) [reply rdtools/normalization.py:93]: I think it makes sense to stay aligned with pvlib

**Comment by mdeceglie** [comment]
Open question for me is whether to leave `normalized_yield` as the parameter for "PV yield data that is normalized, filtered and aggregated" in the plotting module. It is somewhat of a unique quantity so I am leaning toward leaving it as is.

**Comment by cdeline** [comment]
`normalized_yield`  works just fine.  I'd advocate not spending much more time on this particular exercise, we've got bigger fish to fry.  🔥 🐟 🔥

**Comment by cdeline** [comment]
In the notebook there are still two questionable changes that COULD be made.  First: we still have a RdTools function 'get_clearsky_tamb'.  This makes it awkward to rename the 'Tamb' parameter to 'temperature_air'.  Second - in the original notebook we just call the power channel 'power' instead of 'power_dc'.  I kept this as-is since the source file from DKCAS doesn't distinguish whether it's power_dc or power_ac.  I figure this can be left ambiguous.

**Comment by cdeline** [comment]
OK, notebooks have been tested and are passing.  Note that I needed to register the matplotlib converters before plotting time series because pandas > 1,0 doesn't do it automatically, and the fix that matplotlib includes in the 3.3.0 release is useless.

**Comment by cdeline** [comment]
Updated docs and release notes.  Notebooks are passing, I'm merging this per earlier communication with @mdeceglie

**Comment by mdeceglie** [comment]
Thanks @cdeline! Looks great.

**Comment by cdeline** [comment]
Fixes #175

**Pull request merged**